src/components/list/partials/list-card.vue

```diff
@@ -5,7 +5,7 @@
 			'has-background': blurHashUrl !== '' || background !== null,
 		}"
 		:style="{
-			'background-color': list.hexColor,
+			'border-left': `1rem solid ${list.hexColor}`,
```

Comment:
Please only set the border-color as an inline-style and move the rest to the stylesheet.

I also think 1rem is too much.

Reply:
Ok, I left only the ``border-left-color`` as inline style. I also reduced the border to 0.8rem (even though I liked it more before)